config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   LISTENING_PORT: process.env.PORT || 3000,
-  DB_TYPE: "postgres", //mongo or postgres
+  DB_TYPE: "mongo", //mongo or postgres
```

Comment:
read it from env file.
```suggestion
  DB_TYPE: process.env.DB_TYPE || "mongo", //mongo or postgres
```